src/gui/transferlistwidget.cpp

```diff
@@ -696,6 +697,7 @@ void TransferListWidget::displayColumnHeaderMenu()
             continue;
 
         const auto columnName = m_listModel->headerData(i, Qt::Horizontal, Qt::DisplayRole).toString();
+        const auto columnToolTip = m_listModel->headerData(i, Qt::Horizontal, Qt::ToolTipRole);
```

Comment:
```suggestion
        const QVariant columnToolTip = m_listModel->headerData(i, Qt::Horizontal, Qt::ToolTipRole);
```

Reply:
Done. Unfortunately, an approval has been dismissed by pushing updated changes that were rebased onto the master branch.